src/components/Help/Documentation/index.tsx

```diff
@@ -0,0 +1,217 @@
+                  <Link
+                    href="//www.ebi.ac.uk/interpro/api/static_files/swagger/"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Swagger API documentation
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    to={{
+                      description: {
+                        main: { key: 'result' },
+                        result: { type: 'download' },
+                      },
+                      hash: '/entry/interpro/|json',
+                    }}
+                  >
+                    Code snippet generator
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    href="//www.ebi.ac.uk/data-protection/privacy-notice/embl-ebi-public-website"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Privacy and security
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    href="//www.ebi.ac.uk/ebisearch/documentation"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    EBI Search documentation
+                  </Link>
+                </li>
+              </ul>
+            </div>
+            <div className={f('flex-box')}>
+              <span
+                className={f('icon', 'icon-common', 'xl')}
+                data-icon="&#xf10d;"
+              />{' '}
+              <h5>Publications</h5>
+              <ul>
+                <li>
+                  <Link
+                    href={`${config.root.readthedocs.href}citing.html`}
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Citing InterPro
+                  </Link>
+                </li>
+                <li>
+                  <Link
+                    href="https://pfam-docs.readthedocs.io/en/latest/citing-pfam.html"
+                    className={f('ext')}
+                    target="_blank"
+                  >
+                    Citing Pfam
+                  </Link>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div className={f('vf-grid__col--span-1')}>
+          <div className={f('box-add', 'margin-top-xxlarge')}>
+            <h3 className={f('light')}>Contact us/Additional help</h3>
+            <ul>
+              {
+                // <li>
+                //   <Link href="//www.ebi.ac.uk/support/interpro-general-query">
+                //     <span
+                //       className={f('icon', 'icon-common')}
+                //       data-icon="&#x27a;"
+                //     />{' '}
+                //     Chat with an assistant{' '}
+                //   </Link>
+                // </li>
+              }
```

Comment:
What is that? :raised_eyebrow:

Reply:
Not sure, it was in the .js file as well.

Reply:
You can remove it. We do not plan to offer live support, nor we plan to create an AI assistant.